### PR TITLE
Spelling/Phrasing Fix, Missing Links

### PR DESCRIPTION
**Pull request by DBaack11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,8 +133,7 @@ Now it is time to create a solidity smart contract.
     }
    ```
 
-   - And that's it! your code
-     like [this](https://github.com/BlockDevsUnited/BasicFrontEndTutorial/blob/master/contracts/mood.sol)
+   - And that's it! Your code should look like [this]. (https://github.com/BlockDevsUnited/BasicFrontEndTutorial/blob/master/contracts/mood.sol)
```
Comment (haardikk21) [inline README.md]: Can you remove the full-stop here?
[Text](HTTP Link) is markdown syntax for clickable links. The full stop will mess with that syntax.

File: README.md [hunk 2]
```diff
@@ -91,9 +91,9 @@ We will create an app that simply reads and writes a value to the blockchain. We
 
 Now it is time to create a solidity smart contract.
 
-1. You can use any editor you like to make the contract. For this tutorial we recommend the online IDE [remix.ethereum.org]
+1. You can use any editor you like to make the contract. For this tutorial we recommend the online IDE [https://remix.ethereum.org]
```
Comment (haardikk21) [inline README.md]: Might be worth it to just make this a Markdown link as well. Something like

`[Remix](https://remix.ethereum.org)`

File: README.md [hunk 3]
```diff
@@ -91,9 +91,9 @@ We will create an app that simply reads and writes a value to the blockchain. We
...
    - Never used remix before? Checkout [This video](https://www.youtube.com/watch?v=pdJttvcAV1c)
-2. Go to remix.ethereum.org
+2. Go to https://remix.ethereum.org
```
Comment (haardikk21) [inline README.md]: Same as above^

**Review by haardikk21 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (DBaack11) [reply README.md]: Thanks for the feedback, made the changes 👍

**Pull request merged**